### PR TITLE
Rounding errors for roundMode 1 specific cases

### PR DESCRIPTION
**Pull request by IonutBostan**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.6.1.x, 1.7.0.x, 1.7.1.x
| Description?  | If roundMode is 1 then for formatCurrency(77.35000000001, 2, 'lei', 1) the result is 77.34 and it should be 77.35;
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? |no
| Fixed ticket? | 
| How to test?  | Test in javascript console in a product page for roundMode 1  formatCurrency(77.35000000001, 2, 'lei', 1) should return 77.35

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/7859)

### CODE REVIEW

File: js/tools.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ function formatCurrency(price, currencyFormat, currencySign, currencyBlank)
 {
   // if you modified this function, don't forget to modify the PHP function displayPrice (in the Tools.php class)
   var blank = '';
-  price = parseFloat(price.toFixed(10));
+  price = parseFloat(price.toFixed(14));
```
Comment (eternoendless) [inline js/tools.js]: Why 14, specifically?

**Review by eternoendless — COMMENTED** [review]
Hi @IonutBostan! Thanks for this PR.

**Comment by colinegin** [comment]
Hello @PrestaShop/prestashop-core-developers is this PR still needed with the CLDR integration and currency customization ?

**Comment by PierreRambaud** [comment]
I think no because this function is deprecated and will be removed.
https://github.com/PrestaShop/PrestaShop/pull/7859/files#diff-c0719e175dc09e1b42250ebc5380ba51R82

**Comment by colinegin** [comment]
Ok, I close it then. Thank you for your contribution !
Feel free to reopen it if you think this is still needed.

Have a nice day,

Coline

**Pull request closed without merging**